manifests/cf-manifest/manifest/050-rds-broker.yml

```diff
@@ -37,6 +36,8 @@ jobs:
         job: rds_broker
         tags: (( inject meta.datadog_tags ))
       rds-broker:
+        skip_final_snapshot: false
```

Comment:
is this now a global parameter? was it ignored before? what is a default if the per-plan parameter is removed?